kornia/geometry/calibration/distort.py

```diff
@@ -67,6 +70,7 @@ def distort_points(points: torch.Tensor, K: torch.Tensor, dist: torch.Tensor) ->
         dist: Distortion coefficients
             :math:`(k_1,k_2,p_1,p_2[,k_3[,k_4,k_5,k_6[,s_1,s_2,s_3,s_4[,\tau_x,\tau_y]]]])`. This is
             a vector with 4, 5, 8, 12 or 14 elements with shape :math:`(*, n)`.
+        new_K: New intrinsic camera matrix with shape :math:`(*, 3, 3)`. Default: None, in this case K is used.
```

Comment:
Add more explanation like in the OpenCV function, maybe just rephrase that one. Btw, I see they use identity instead

Reply:
I will try but explanation in opencv is ultra bad. They are doing the same as in this code.

Reply:
I found this description, I think is the best one: "Camera matrix of the distorted image. By default, it is the same as cameraMatrix but you may additionally scale and shift the result by using a different matrix."